mmv1/templates/terraform/examples/integration_connectors_connection_basic.tf.erb

```diff
@@ -0,0 +1,14 @@
+resource "google_integration_connectors_connection" "<%= ctx[:primary_resource_id] %>" {
+  name     = "<%= ctx[:vars]['connection_name'] %>"
+  connector_version = "projects/connectors-harsha/locations/global/providers/gcp/connectors/pubsub/versions/1"
```

Comment:
I'm assuming this is a resource located in your project? I wonder if we can create a resource in the tests or linked to a publicly accessible resource?

Reply:
Changed, this is a project specific resource, so I have created a new project - connectors-example and used it here.

Reply:
Can we create such resource within the test. We can't have dependency on resources that we don't have control of.